nixos/modules/hardware/fw-fanctrl.nix

```
@@ -0,0 +1,125 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.hardware.fw-fanctrl;
+  defaultConfig = lib.importJSON "${pkgs.fw-fanctrl}/share/fw-fanctrl/config.json";
+in
+{
+  options.hardware.fw-fanctrl = {
+    enable = lib.mkEnableOption "Enable fw-fanctrl systemd service and install the needed packages.";
```

```suggestion
    enable = lib.mkEnableOption "the fw-fanctrl systemd service and install the needed packages";
```

see https://github.com/NixOS/nixpkgs/blob/master/lib/options.nix#L187